lib/chef/resource/chocolatey_installer.rb

```diff
@@ -0,0 +1,198 @@
+class Chef
+  class Resource
+    class ChocolateyInstaller < Chef::Resource
+      provides :chocolatey_installer
+
+      description "Use the Chocolatey Installer resource to ensure that Choco is installed to your specification. Use the Chocolatey Feature resource to customize your install"
```

Comment:
If that's a resource, use it's name, "Use the`chocolatey_feature` resource..."

If it's not, reword this to be more clear.

Reply:
done